plugins/modules/system/sudoers.py

```diff
@@ -140,7 +154,8 @@ def content(self):
 
         commands_str = ', '.join(self.commands)
         nopasswd_str = 'NOPASSWD:' if self.nopassword else ''
-        return "{owner} ALL={nopasswd} {commands}\n".format(owner=owner, nopasswd=nopasswd_str, commands=commands_str)
+        runas = self.runas
+        return "{owner} ALL=({runas}) {nopasswd} {commands}\n".format(owner=owner, runas=runas, nopasswd=nopasswd_str, commands=commands_str)
```

Comment:
In case `runas` is `None`, this should be exactly the same string as before. Otherwise this is a breaking change.